examples/Firebase_ESP8266_LEDs/Firebase_ESP8266_Neopixel/Firebase_ESP8266_Neopixel.ino

```diff
@@ -25,10 +25,11 @@
 #define PIN 13
 Adafruit_NeoPixel strip = Adafruit_NeoPixel(32, PIN, NEO_GRB + NEO_KHZ800);
 
+#define JSON_BUFFER_SIZE 10*4
 
 // TODO: Replace with your own credentials and keep these safe.
-Firebase fbase = Firebase("brilliant-fire-9638.firebaseio.com")
-                   .auth("llXzJtqwdiHCHw6F6sehQR2EPwbO0YYdhEjPRZxG");
+Firebase fbase = Firebase("YOUR-PROJECT-ID.firebaseio.com")
+                   .auth("YOUR_AUTH_SECRET");
```

Comment:
Maybe use the same placeholder that we use in the other sample:
https://github.com/googlesamples/firebase-arduino/blob/master/examples/FirebasePush_ESP8266/FirebasePush_ESP8266.ino#L23

Or change the other sample to use yours :) (I'm not sure Project ID is in firebase jargon)